images/kamu-base-with-data-mt/init-workspace.py

```diff
@@ -1,48 +1,55 @@
 #!/usr/bin/env python
 
-import os
-import sys
+import shutil
 import subprocess
+from pathlib import Path
 
 ###############################################################################
 
+CURRENT_PATH = Path(__file__).resolve().parent
 S3_REPO_URL = "s3://datasets.kamu.dev/odf/v2/example-mt/"
 
+
 ###############################################################################
 
 def s3_listdir(url):
     return [
         line.strip().split(' ')[1]
         for line in subprocess.run(
-            f"aws s3 ls {url}", 
-            shell=True, 
+            f"aws s3 ls {url}",
+            shell=True,
             text=True,
             check=True,
             capture_output=True,
         ).stdout.splitlines()
     ]
 
+
 def s3_cat(url):
     return subprocess.run(
-        f"aws s3 cp {url} -", 
-        shell=True, 
+        f"aws s3 cp {url} -",
+        shell=True,
         text=True,
         check=True,
         capture_output=True,
     ).stdout.strip()
 
+
 ###############################################################################
 
 subprocess.run(
-    "kamu init --multi-tenant --exists-ok", 
+    "kamu init --multi-tenant --exists-ok",
     shell=True,
     check=True,
 )
 
-for did in s3_listdir(S3_REPO_URL):
+shutil.copy(CURRENT_PATH / "extra/.kamuconfig", ".kamuconfig")
```

Comment:
Note: convenient for both:
- direct script launching 
- CI calling (`cd ./images/ && make kamu-base-with-data-mt`)